src/autocomplete.js

```diff
@@ -93,7 +96,7 @@ export default class Autocomplete extends Component {
   }
 
   isQueryAnOption (query, options) {
-    return options.map(entry => this.templateInputValue(entry).toLowerCase()).indexOf(query.toLowerCase()) !== -1
+    return options.some(entry => (this.templateInputValue(entry) || '').toLowerCase() === query.toLowerCase())
```

Comment:
Correct, except that:

> allowing for `this.templateInputValue(entry)` to return `false`

is more broadly "`this.templateInputValue(entry)` to return something falsey" - and this is the key, it allows us to return undefined from `templates.inputValue`:

```js
templates: {
        inputValue: function(suggestion) {
          return suggestion && suggestion.Title;
        }
}
```

which you have to do if you're providing an `inputValue` function, because it is called when the plugin loads, so not handling an `undefined` `suggestion` argument would result in an Exception.